dealroom_firestore_connector/__init__.py

```diff
@@ -407,6 +407,29 @@ def _get_final_url_and_dealroom_id(
     return final_url, dealroom_id
 
 
+def check_for_deleted_profiles(
+    key_found, history_refs, dealroom_id, count_history_refs
```

Comment:
```suggestion
    doc_refs, dealroom_id, count_history_refs
```
you do not need `key_found` as arg, since the condition to call this function is already checking that that variable has the correct value. what you need here as arg is `doc_refs`, which when you call the function is `doc_refs = history_refs[key_found]`
@GuillermoDR

Reply:
Fixed in [452d4a4](https://github.com/dealroom/data-firestore-connector/pull/28/commits/452d4a4abaa4da12b01b4e7f75877ce1ddf6d3af)